eZ/Publish/Core/FieldType/TextBlock/Type.php

```diff
@@ -119,11 +119,14 @@ protected function checkValueStructure(BaseValue $value)
      *
      * @param \eZ\Publish\Core\FieldType\TextBlock\Value $value
      *
-     * @return array
+     * @return string
      */
     protected function getSortInfo(BaseValue $value)
     {
-        return false;
+        return $this->transformationProcessor->transformByGroup(
+            substr(strtok(trim($value->text), "\r\n"), 0, 255),
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1858/commits/1777d726e8e76f48369dd902d3d443abd9f2530e